java/configuration/src/main/java/sleeper/configuration/properties/UserDefinedInstanceProperty.java

```diff
@@ -977,6 +983,62 @@ public interface UserDefinedInstanceProperty extends InstanceProperty {
             .validationPredicate(Utils::isPositiveInteger)
             .propertyGroup(InstancePropertyGroup.DEFAULT).build();
 
+    UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_MIN_JOB_SIZE = Index.propertyBuilder("sleeper.default.ingest.batcher.job.min.size")
+            .description("Specifies the minimum file size required for an ingest job to be batched and sent. " +
+                    "An ingest job will be created if the batcher runs while this much data is waiting, and the " +
+                    "minimum number of files is also met.")
+            .defaultValue("1G")
+            .validationPredicate(Utils::isValidNumberOfBytes)
+            .propertyGroup(InstancePropertyGroup.DEFAULT).build();
+    UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_MAX_JOB_SIZE = Index.propertyBuilder("sleeper.default.ingest.batcher.job.max.size")
+            .description("Specifies the maximum total file size for a job in the ingest batcher. " +
+                    "If more data is waiting than this, it will be split into multiple jobs. " +
+                    "If a single file exceeds this, it will still be ingested in its own job. " +
+                    "It's also possible some data may be left for a future run of the batcher if some recent files " +
+                    "overflow the size of a job but aren't enough to create a job on their own.")
+            .defaultValue("1G")
+            .validationPredicate(Utils::isValidNumberOfBytes)
+            .propertyGroup(InstancePropertyGroup.DEFAULT).build();
+    UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_MIN_JOB_FILES = Index.propertyBuilder("sleeper.default.ingest.batcher.job.min.files")
+            .description("Specifies the minimum number of files for a job in the ingest batcher. " +
+                    "An ingest job will be created if the batcher runs while this many files are waiting, and the " +
+                    "minimum size of files is also met.")
+            .defaultValue("1")
+            .validationPredicate(Utils::isPositiveInteger)
+            .propertyGroup(InstancePropertyGroup.DEFAULT).build();
+    UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_MAX_JOB_FILES = Index.propertyBuilder("sleeper.default.ingest.batcher.job.max.files")
+            .description("Specifies the maximum number of files for a job in the ingest batcher. " +
+                    "If more files are waiting than this, they will be split into multiple jobs. " +
+                    "It's possible some data may be left for a future run of the batcher if some recent files " +
+                    "overflow the size of a job but aren't enough to create a job on their own.")
+            .defaultValue("10")
```

Comment:
I think this default should be a lot higher given that we're defending against jobs being too big using the total file size. Maybe 100?

Reply:
Done, thanks